src/remote/playwrightServer.ts

```diff
@@ -66,12 +77,12 @@ export class PlaywrightServer {
     server.on('error', error => debugLog(error));
 
     const path = this._delegate.path;
-    const wsEndpoint = await new Promise<string>(resolve => {
+    const wsEndpoint = await new Promise<string>((resolve, reject) => {
       server.listen(port, () => {
         const address = server.address();
         const wsEndpoint = typeof address === 'string' ? `${address}${path}` : `ws://127.0.0.1:${address.port}${path}`;
         resolve(wsEndpoint);
-      });
+      }).on('error', reject);
```

Comment:
Fixes an issue when the port which was provided by the user was already used that it silently failed.